modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -4532,6 +4620,7 @@ def _arrow_type_to_dtype(cls, arrow_type):
         import pyarrow
 
         try:
+            # TODO: should we map arrow types to pyarrow-backed pandas types?
```

Comment:
Why?

Reply:
It seems like this might help avoid the expense of transferring data between backends (numpy and pyarrow), but I'm not sure how this fits into the type inference system in pandas.

Reply:
Let's expand this comment on this matter.